test/base_results/upgrades/timeseries/results_output.csv

```diff
@@ -1727,9 +1727,9 @@ project_national,0.0,0.000511821,0.0020131625999999995,0.0,0.0,0.001228370399999
 project_national,0.0,0.00037533539999999994,0.001364856,0.0,0.0,0.0,0.0,0.0007165494,0.001023642,0.0,0.003992203799999999,0.0,0.0,0.0017743128,0.0,0.00784,0.0,0.0018084341999999999,0.0093492636,0.00784,0.0,0.0,0.0
 project_national,0.0,0.0,0.0011260062,0.0,0.0,0.0,0.0,0.0004776996,0.001023642,0.0006141851999999999,0.003992203799999999,0.0,0.0,0.0014672201999999998,0.0,0.01475,0.0,0.0018084341999999999,0.009212778,0.01475,0.0,0.0,0.0
 project_national,0.0,6.824279999999999e-05,3.4121399999999995e-05,0.0,0.0,0.0,0.0,0.0004776996,0.001023642,0.0,0.003992203799999999,0.0,0.0,0.0014672201999999998,0.0,0.01726,0.0,0.0014672201999999998,0.007063129799999999,0.01726,0.0,0.0,0.0
-project_national,0.0,6.824279999999999e-05,0.0002388498,0.0,0.0,0.0,0.0,0.0004776996,0.001023642,0.0,0.003992203799999999,0.0,0.0,0.0014672201999999998,0.0,0.0,0.0,0.0015695843999999999,0.007336100999999999,0.0,0.0,0.0,0.0
-project_national,0.0,6.824279999999999e-05,0.00020472839999999997,0.0,0.0,0.0,0.0,0.0004776996,0.001023642,0.0,0.003992203799999999,0.0,0.0,0.0014672201999999998,0.0,0.0077,0.0,0.0015695843999999999,0.0073019795999999994,0.0077,0.0,0.0,0.0
-project_national,0.0,6.824279999999999e-05,0.000170607,0.0,0.0,0.0,0.0,0.0004776996,0.001023642,0.0,0.003992203799999999,0.0,0.0,0.0014672201999999998,0.0,0.0,0.0,0.001535463,0.007199615399999999,0.0,0.0,0.0,0.0
+project_national,0.0,6.824279999999999e-05,0.00020472839999999997,0.0,0.0,0.0,0.0,0.0004776996,0.001023642,0.0,0.003992203799999999,0.0,0.0,0.0014672201999999998,0.0,0.0,0.0,0.001535463,0.0073019795999999994,0.0,0.0,0.0,0.0
+project_national,0.0,6.824279999999999e-05,0.000170607,0.0,0.0,0.0,0.0,0.0004776996,0.001023642,0.0,0.003992203799999999,0.0,0.0,0.0014672201999999998,0.0,0.0077,0.0,0.001535463,0.007267858199999999,0.0077,0.0,0.0,0.0
+project_national,0.0,6.824279999999999e-05,0.00013648559999999998,0.0,0.0,0.0,0.0,0.0004776996,0.001023642,0.0,0.003992203799999999,0.0,0.0,0.0014672201999999998,0.0,0.0,0.0,0.001535463,0.007199615399999999,0.0,0.0,0.0,0.0
```

Comment:
@joseph-robertson Looks like we're getting CI diffs because the order is changing; can we prevent that? (And how about using fewer decimal places for these outputs?)

Reply:
@aspeake I wonder if this is a DST issue. (I'll look into the decimal places issue.)